n64.ld

```diff
@@ -22,22 +22,24 @@ OUTPUT_ARCH (mips)
 EXTERN (_start)
 ENTRY (_start)
 
+__mem_start = 0x80000000;
+
+/* Start address of code is 1K up in uncached, unmapped RAM.  We have
+* to be at least this far up in order to not interfere with the cart
+* boot code which is copying it down from the cart
+*/
+__libdragon_code_start = __mem_start + 0x400;
```

Comment:
This instead makes sense to have as a linker-defined symbol after all. Maybe `__libdragon_text_start` is a little bit more clear, given that we're speaking of a linker symbol?

Reply:
There is also a `__text_start` that was the reason for the name but that is fine as well. I'll update.